scripts/martech.js

```diff
@@ -0,0 +1,41 @@
+import { getMetadata, loadScript } from './aem.js';
+// eslint-disable-next-line import/no-cycle
+import { getLocaleAsBCP47 } from './scripts.js';
+
+function initDataLayer() {
+  window.digitalData = {
+    page: {
+      pageInfo: {
+        brand: 'sid',
+        pageName: getMetadata('og:title'),
+        destinationURL: window.location.href,
+        referringURL: document.referrer,
+        language: getLocaleAsBCP47(),
+        pageURL: window.location.href,
+        pageType: 'homepage', // TODO: parse pagetype dynamically?
+        pageDescription: getMetadata('og:description'),
+        country: 'US', // TODO: fetch country dynamically?
```

Comment:
We can just grab the country code from the URL path, e.g. /en-**us**/

Reply:
Yes, but this still needs a custom conversion to the correct country code format. We can add that later as we will only serve the us site for now. But shouldn't be a huge task to do so anyway. Just wanted to get these changes out as quickly as possible so testing can progress